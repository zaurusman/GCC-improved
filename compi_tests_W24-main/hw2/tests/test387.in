while (

		
 
	 	
   

	


 	8
)
 
	 
 		

  

   	 		
 

		 




		
	 			
j 		 	 =

     false/ 
 

 	

 		 				

	"harry"- 4and
not		  		
	 
2541865828329   			   
	   

	/
		 		

 

 	 		 not (5 ) and true	or	 
			
   




 yand false;	 

		
while(i(true) )continue; bytex;//Your laptopissmallerthenrequiredtopass thesetests.:
	 				  			
(return;if( 
 
6
 
 	 	

b)


	  
	   	

inta ;//Yourpp isnoneexistent...
 	
inty=not notfalse*
0	
b+trueandtrue  


 		

	-false

 
	

		
		  
 	
   
	
	 
	 

	
		
 
 
 
 	or 
	

 false / 		( bool 	 	
 	 	)(		 			
byte)"\"" andtrue	 ;//Yourgithub	isun				 		  

 	
 		
 				   
	
 
 
	 
  readable.^^while(9 b	
  
		 	

)
int j;

break	 ;break;continue;	
 	boola=notfalse*3	b;//Yoursexlife	  
 




 
	 
   


	
 		
soFATwhenshejumps
  
	into the	
 	 oceanthe	oceanjumpsout...continue;
	
			
 

   
 	

 		 

 	//Your githubshould go back	to					

  		 	


  first
 
 

 
 	
	




 
grade.^^inta =not "a wizard"* y
	
	
		 	
	or8b+(int  	 
 	
) (bool)
  		

	 aand7 b-
	 
	 			



  	9b!=x(( 
int
 

	)true*	 

	5-	true+
false+"harry"	
 ) *	"\t"and2;//Your
	
	  	
	 
	
code is 
 


smaller thenrequired topassthese tests. :)

		 		
		 			


 
		
		 		

	   	
	
  

	 
   	  
		 
bool
 
 	
 i =	 



 true -false	*	 	
 	
  	   

	
 	
	
 

	 	
 	
a(
falseortrue)
*true;
while ( 		7 )boolx;while	
	




	
		 


	  
		
 ("c" )
continue;
if
  	
(100000000000000 b
 	) 	
 	
   
  	
			

	
 	{if(
   


  

   		  		  	 	
 1	
)j
	
(y(xor6));elsebytey	="b"+false; 	
 		  
	
  
     

}	 	 	els`e
continue;//JYour
code

 
 

 	  	




 
 
	
	

 is


 
  un& readable. uwuy(not0b

	)R;

	

  	by^te
		
	
 

j 			 ;
j=7b and
9b<=\true		
-7b;//Yourpa5rtner 
	
		

 			m1akesrecrutersstart 	smoking<3
jj(
 
truBe);
				 	
 boolz=a;return
		
	

	 		

;


 
  
	

 				
	  		 
 
	
 		
 

	
	return;