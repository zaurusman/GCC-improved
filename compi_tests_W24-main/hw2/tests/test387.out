23: Exp -> NUM
27: Exp -> FALSE
25: Exp -> STRING
20: Exp -> Exp BINOP Exp
23: Exp -> NUM
20: Exp -> Exp BINOP Exp
23: Exp -> NUM
28: Exp -> NOT Exp
23: Exp -> NUM
19: Exp -> LPAREN Exp RPAREN
28: Exp -> NOT Exp
20: Exp -> Exp BINOP Exp
29: Exp -> Exp AND Exp
26: Exp -> TRUE
29: Exp -> Exp AND Exp
21: Exp -> ID
30: Exp -> Exp OR Exp
line 56: syntax error
